pkg/jerryscript/Makefile.include

```diff
@@ -0,0 +1 @@
+INCLUDES += -I$(BINDIRBASE)/pkg/$(BOARD)/jerryscript/jerry-core/
```

Comment:
Then please investigate. The above include is incorrect. It should break at least Murdock2.